### DISCUSSION TITLE
Template

### DISCUSSION THREAD

**Discussion by gregdon** [microsoft/fluentui#28415] [Q&A]
I’m looking to propose Fluent-UI as a solution for my company which is a large MS shop… This will be a little easier if I had access to a template - think a template like an admin dashboard with header, footer, aside (drawer), main content…. Much like what you can find at themeforest.  Has anyone put a base template together that they are willing to share?

**Comment by behowell** [comment]
Hi @gregdon, I'm not aware of a template like what you're asking for, sorry. 

Please see https://fluent2.microsoft.design/ for details about how to get started with FluentUI, and the components available. The individual component pages also have links to our [storybook](https://react.fluentui.dev), which has more examples for each component.
